docs/src/pages/demos/breadcrumbs/CustomizedBreadcrumbs.tsx

```diff
@@ -9,57 +8,46 @@ import Avatar from '@material-ui/core/Avatar';
 import HomeIcon from '@material-ui/icons/Home';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 
-interface CustomBreadcrumbProps extends WithStyles<typeof styles> {}
+const StyledBreadcrumb = withStyles((theme: Theme) => ({
+  root: {
+    backgroundColor: theme.palette.grey[100],
+    height: 24,
+    color: theme.palette.grey[800],
+    fontWeight: theme.typography.fontWeightRegular,
+    '&:hover, &:focus': {
+      backgroundColor: theme.palette.grey[300],
+    },
+    '&:active': {
+      boxShadow: theme.shadows[1],
+      backgroundColor: emphasize(theme.palette.grey[300], 0.12),
+    },
+  },
+}))(Chip) as typeof Chip; // need the cast here because https://github.com/Microsoft/TypeScript/issues/26591
 
-interface CustomizedBreadcrumbsProps extends WithStyles<typeof styles> {}
+function handleClick(event: React.MouseEvent<Element, MouseEvent>) {
```

Comment:
@oliviertassinari This was assigned to components that attached this to an `a` (HTMLAnchorElement) element which does not match `HTMLDivElement` (don't ask me what difference their interface has). Since we're only interested in `preventDefault` we can be a bit more generous.